jetty-fcgi/fcgi-server/src/main/java/org/eclipse/jetty/fcgi/server/HttpChannelOverFCGI.java

```diff
@@ -152,10 +167,7 @@ protected boolean eof()
     {
         if (LOG.isDebugEnabled())
             LOG.debug("received EOF");
-        try (AutoLock l = _lock.lock())
-        {
-            _specialContent = new HttpInput.EofContent();
-        }
+        _specialContent = new HttpInput.EofContent();
```

Comment:
`new HttpInput.EofContent()` could be a constant.

Reply:
Done.